tests/Uri/test_uri.py

```diff
@@ -0,0 +1,56 @@
+import pytest
+from smsdk import client
+from smsdk.utils import get_url
+
+
+def test_uri() -> None:
+    tenant: str = ""
+    cli: client.Client = client.Client(tenant)
+
+    assert cli.tenant == tenant
```

Comment:
Is empty string tenant valid?  Should this be an error?

Reply:
Add a comment to that effect.  I think it is a bug if the CLI does not assume a default hostname.

Reply:
I am leaning towards making it an error as a test of it 

```
ENV_SDK_VAR_API_KEY=junk ENV_SDK_VAR_API_SECRET=junk ENV_SDK_VAR_TENANT="" pytest -vvv -s tests/factory/test_factory.py
============================= test session starts ==============================
platform linux -- Python 3.10.12, pytest-7.2.1, pluggy-1.5.0 -- /home/mklein0/VirtualEnvs/sdk/bin/python
cachedir: .pytest_cache
rootdir: /home/mklein0/Repositories/git/sightmachine/Code/sightmachine-sdk, configfile: pytest.ini
collecting ... collected 2 items

tests/factory/test_factory.py::test_get_utilities PASSED
tests/factory/test_factory.py::test_get_factories Failed login attempt to https://.sightmachine.io. Check tenant name, secret, and key
Error getting data, but continuing. URL has an invalid label.
Error getting data, but continuing. URL has an invalid label.
...
```

Reply:
Looks like the tests do not verify the login result.  I would not be surprised ot developers forget this as well.
https://github.com/sightmachine/sightmachine-sdk/blob/2a46c6b0d75cfe508bfd3cbe618d0145646e75d7/tests/conftest.py#L22-L26

Need
```python
    assert cli.login()
```

Reply:
Added assertion on the login result